src/Directory.Build.props

```diff
@@ -0,0 +1,77 @@
+<Project>
+	<!-- Common packages -->
```

Comment:
I don't think these are all required by this project.

Reply:
No they are not. But since dependabot is still broken, at least that way it will only mess up one file.

Reply:
ah, the GitHub interface made it look like this file was in the Bufdio.Spice86 directory.

But I don't feel this is the right way to solve issues with dependabot.
All of the projects now have unused dependencies...

We can look into Dependabot alternatives (at work I use Renovate). Or we can just manually click the "Upgrade packages in solution" button in Rider once in a while.

Reply:
I'll put non common packages werr they belong so we can move on with this PR. 😊

Yeah replacing dependabot sounds good.

Reply:
https://github.com/apps/renovate

Reply:
I re-did the PR, now Directory.Build.props only contains common packages (Sourcelink, source code analyzers)